docs/scos/dev/back-end-development/project-namespaces.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Project namespaces
```

Comment:
Also my points:
okay, following the original thread from Denis, this one is important:
`safari includes mostly usage of Pyz, but now why do we use and when to create other project namespaces.`

What’s missing in the document:
* When and why introducing a new namespace (in parallel to Pyz) is benefitial
* How to introduce a new namespace (app configuration, new folder, PSR adjustment in composer.json, adjustments to tests for a new Namespace)
* How does it work, in terms of class inheritance